src/components/Curl/Curl.tsx

```diff
@@ -0,0 +1,79 @@
+import { MDXProvider } from '@mdx-js/react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import * as React from 'react';
+
+import { pre as Pre } from '../Mdx';
+
+import styles from './Curl.module.scss';
+
+interface ICurl {
+  request: string;
+  response: any;
+}
+
+const Curl: React.FC<ICurl> = (props) => {
+  const { request, response } = props;
+  return (
+    <MDXProvider
+      disableParentContext
+    >
+      <div
+        className={styles.container}
+      >
+        <Pre
+          aria-label="Request"
+          className={classNames(
+            'language-bash',
+            styles.request,
+          )}
+          showLineNumbers={false}
+        >
+          <code
+            className="language-bash"
+          >
+            {request}
+          </code>
+        </Pre>
+        <Pre
+          aria-label="Response"
+          className={classNames(
+            'language-json',
+            styles.response,
+          )}
+          showLineNumbers={false}
+        >
+          <code
+            className="language-json"
+          >
+            {`< HTTP/2 200
+< content-type: text/html; charset=utf-8
+< set-cookie: __cfduid=d5180fc96c14bfdf800067d787a77baae1613692378; expires=Sat, 20-Mar-21 23:52:58 GMT; path=/; domain=.wesrice.com; HttpOnly; SameSite=Lax
```

Comment:
We should remove the `set-cookie`, `x-*`, and `cf-ray` headers from this

Reply:
This component was an idea I was playing with. I think it'd be nice to have a component for request/response examples that stacks two codeblocks – a code example for a curl command w/ the json body and then another for the response.

I've saved this in another branch and might bring it back in the response content pr.